test/Conversion/TorchOnnxToTorch/simple_ops_a_to_f.mlir

```diff
@@ -1020,3 +1020,116 @@ func.func @ints_constant() -> !torch.vtensor<[2], si64> attributes {torch.onnx_m
   return %0 : !torch.vtensor<[2],si64>
 }
 
+// CHECK-LABEL: @test_flatten_4d_axis_2
+func.func @test_flatten_4d_axis_2(%arg0: !torch.vtensor<[2,3,4,5],f32>) -> !torch.vtensor<[6,20],f32> attributes {torch.onnx_meta.ir_version = 7 : si64, torch.onnx_meta.opset_version = 13 : si64, torch.onnx_meta.producer_name = "backend-test", torch.onnx_meta.producer_version = ""} {
+  // CHECK: %[[RIGHT_START:.*]] = torch.constant.int 2
```

Comment:
A small nit. These tests will fail if the IR gets printed as 

torch.constant.int 3
torch.constant.int 2

instead of 

torch.constant.int 2
torch.constant.int 3

This could in theory happen in something in MLIR upstream changes or someone changes the onnx->torch-mlir code you've implemented here for whatever reason. 

If it doesn't matter in what order 2 lines appear, you can use CHECK-DAG. So  

// CHECK-DAG: torch.constant.int 3
// CHECK-DAG: torch.constant.int 2

will match both of the above.

Reply:
@newling Thanks for the review, James!  Yes, I'm aware of the potential for reordering, as I've encountered it in the past. I agree that if the -convert-torch-onnx-to-torch pass gets more complicated and isn't just the rewrites here, then the ordering could be changed.  However, all the other torch-onnx tests are relying on order (that is, I don't see any CHECK-DAGs anywhere), so the first person to try such a thing will have fun cleaning up all the tests!  :-D  I take the reliance on order preservation to mean that we intend not to add fancy order-changing optimizations to this pass.

Reply:
Better to improve practice now and fix up other tests later, so I would change to `CHECK-DAG`. MLIR does not guarantee to maintain operation order between passes so independent of our own changes it is possible that the operation ordering could change.

Reply:
@newling @rsuderman  Now using CHECK-DAG for all lines that could potentially appear in any order: not just the constants but the first (RHS) collapse intermediate results as well.  Lit tests (still) pass.